printers.tex

```diff
@@ -22,19 +24,21 @@
 
 \begin{verbatim}
 >>> pprint(Integral(sqrt(phi0 + 1), phi0), use_unicode=False)
-  /
- |
- |   __________
+  /                    
+ |                     
+ |   __________        
```

Comment:
Yes, this was changed before `NORMALIZE_WHITESPACE` option was given. Now, these spaces can be removed